apps/ai-image-generator/backend/test/scripts/call-hosted-app-action.ts

```diff
@@ -0,0 +1,144 @@
+import { AppActionProps, PlainClientAPI, createClient } from 'contentful-management';
+import { WebhookCallDetailsProps } from 'contentful-management/dist/typings/entities/webhook';
+import { parseArgs } from 'node:util';
+import util from 'util';
+
+class AppActionRunner {
```

Comment:
I'd feel better if this class was it's own file and then this file was strictly for the main function that runs

Reply:
Yeah I was thinking we could at some point extract this to potentially do some kind of integration testing. But the class itself is not really relevant to any other part of this code base, so maybe I'll defer on this particular point to see how our use of this script evolved (or doesn't).

In general I just made it a class to make it easier for my brain to organize things.